recipes/nvidia-apex/LICENSE

```diff
@@ -0,0 +1,11 @@
+All rights reserved.
```

Comment:
Make sure to make a PR to get them to package this upstream.

Reply:
@beckermr my bad - it already is packaged in the github repo.

Reply:
no worries! simply delete this file and make sure the name of the license matches what is in the recipe!